cmd/webhook/main.go

```diff
@@ -37,6 +41,22 @@ import (
 	"knative.dev/pkg/webhook"
 )
 
+const (
+	RegistrationDelayTime = "REG_DELAY_TIME"
+)
+
+func getRegistrationDelayTime() time.Duration {
+	var RegistrationDelay time.Duration
```

Comment:
If the env variable has not been specified, will this default to 0? If so, seems like a possibly bad timeout? :) Perhaps, we should explicitly initialize to a default value in case env variable has not been specified?

Reply:
Alternatively, the other variables (L106+) will panic if the env has not been specified. I think it would be better to not panic but default, but thoughts?

Reply:
The default value of `RegistrationDelay` is 0 if the env variable has not been specified.  So If we don't set the env
```
        - name: REG_DELAY_TIME
          value: "10"
```
the behavior will be the same as before.  0 is the default value. Do we really need to explicitly initialize to a default value? @vaikas-google

Reply:
well, perhaps I misunderstood the bug. My reading of the bug is that by having it be effectively 0 (what it was before) was causing a bug then without a default that's not 0, if not specified it will fail. I would be better for a user if we provided a value that's less likely to fail?

Reply:
I see. I create a knative-eventing chart, only use the chart to deploy `knative` can lead the bug https://github.com/knative/pkg/issues/417. The current deploy method can run normally if we just follow the knative tutorial use `kubectl apply -f xxxx.yaml`. So I think the 0 as the default value is OK. Thanks @vaikas-google